mkdocs.yaml

```
@@ -46,6 +46,7 @@ extra_css:
 extra:
   generator: false
   environment: !ENV [DOC_ENV, "dev"]
+  wp_options_api_url: !ENV WP_OPTIONS_API_URL
```

@jmfontaine please suggest how could we pass this env var from GH action?

---

We could pass that as a GitHub secret as we do for `DOC_ENV`. What would be the value for the production environment? 

Also, could you please provide a default value because a lack of value breaks the local development environment?

Here is my suggestion:

```suggestion
  wp_options_api_url: !ENV [WP_OPTIONS_API_URL, "https://spaceliftio.wpcomstaging.com/wp-json/acf/v3/options/options"]
```

I used the value you had hardcoded previously but feel free to change it as you see fit.

---

I added the value to the repository's `WP_OPTIONS_API_URL` env var.